inc/saiswitch.h

```diff
@@ -2695,6 +2695,23 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_IPSEC_SA_STATUS_CHANGE_NOTIFY,
 
+    /**
+     * @brief Number of ECMP members supported by My MAC entries installed on the switch
+     *
+     * @type sai_uint32_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_MAX_ECMP_MEMBER_COUNT,
```

Comment:
what's the different vs existing attribute 

    /**
     * @brief ECMP number of members per group
     *
     * Default is 64
     *
     * @type sai_uint32_t
     * @flags READ_ONLY
     */
    SAI_SWITCH_ATTR_ECMP_MEMBERS,

Reply:
Existing attribute SAI_SWITCH_ATTR_ECMP_MEMBERS is used in conjunction with the boot time SAI_KV_NUM_ECMP_GROUPS and is read only. We can not configure dynamically the ECMP_MEMBER size based on HW capability